test_runner/src/main/kotlin/ftl/log/OutputLogger.kt

```diff
@@ -0,0 +1,20 @@
+package ftl.log
+
+fun setLogLevel(logLevel: OutputLogLevel) {
+    minimumLogLevel = logLevel
+}
+
+fun log(message: Any, level: OutputLogLevel = OutputLogLevel.SIMPLE) =
+    if (minimumLogLevel >= level) print(message)
+    else Unit
+
+fun logLn(message: Any = "", level: OutputLogLevel = OutputLogLevel.SIMPLE) =
+    if (minimumLogLevel >= level) println(message)
+    else Unit
```

Comment:
We do not return need output from this method so I suggest to 

```kotlin
fun log(message: Any, level: OutputLogLevel = OutputLogLevel.SIMPLE) {
    if (minimumLogLevel >= level) print(message)
}

fun logLn(message: Any = "", level: OutputLogLevel = OutputLogLevel.SIMPLE) {
    if (minimumLogLevel >= level) println(message)
}
```

Reply:
done